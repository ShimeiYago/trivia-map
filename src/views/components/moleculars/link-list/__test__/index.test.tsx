import { shallow, ShallowWrapper } from 'enzyme';
import { LinkList, Props } from '..';

let wrapper: ShallowWrapper<Props>;

const props: Props = {
  list: [
    {
      icon: 'x',
      text: 'text',
      link: 'https://xxx',
    },
  ],
};

describe('Shallow Snapshot Tests', () => {
  beforeEach(() => {
    wrapper = shallow(<LinkList {...props} />);
  });

  it('basic', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
