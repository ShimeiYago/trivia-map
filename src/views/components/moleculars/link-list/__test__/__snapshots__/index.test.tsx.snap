// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Snapshot Tests basic 1`] = `
<ForwardRef(List)>
  <NonStyleLink
    key="account-setting-0"
    to="https://xxx"
  >
    <ForwardRef(ListItem)>
      <ForwardRef(ListItemButton)>
        <ForwardRef(ListItemIcon)>
          x
        </ForwardRef(ListItemIcon)>
        <ForwardRef(ListItemText)
          primary="text"
        />
        <ForwardRef(ListItemIcon)>
          <Memo(ForwardRef(KeyboardArrowRightIcon)) />
        </ForwardRef(ListItemIcon)>
      </ForwardRef(ListItemButton)>
    </ForwardRef(ListItem)>
  </NonStyleLink>
</ForwardRef(List)>
`;
